src/server/models/PartiesModel.js

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs';
+import path from 'path';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+
+export default class Parties {
+  constructor(id, userId, name, hqAddress, logoUrl) {
+    this.id = id;
+    this.userId = userId;
+    this.name = name;
+    this.hqAddress = hqAddress;
+    this.logoUrl = logoUrl; 
+  }
+
+  save() {
+    return new Promise((resolve, reject) => {
+      setTimeout(() => {
+        const newInstance = {
+          id: this.id,
+          userId: this.userId,
+          name: this.name,
+          hqAddress: this.hqAddress,
+          logoUrl: this.logoUrl,
+        };
+        const file = fs.readFileSync(
+          path.join(__dirname, '../datastore/index.json'), 'utf8',
+        );
+        const db = JSON.parse(file);
+        db.party.push(newInstance);
+        fs.writeFileSync(
+          path.join(__dirname, '../datastore/index.json'),
+          JSON.stringify(db), (err) => {
```

Comment:
This method is going to persist party data. Rename your datastore to normal javascript file instead of json, so that you will use `array.push()` method to save data